Tools/CISupport/ews-build/steps.py

```diff
@@ -7161,31 +7160,10 @@ def run(self):
             yield self._addToLog('stdio', 'No API key for {} found'.format(RESULTS_DB_URL))
 
         self.command = ['python3', 'Tools/Scripts/compare-static-analysis-results', os.path.join(self.getProperty('builddir'), 'build/new')]
-        self.command += ['--build-output', SCAN_BUILD_OUTPUT_DIR]
-        if not self.use_expectations:
-            self.command += ['--archived-dir', os.path.join(self.getProperty('builddir'), 'build/baseline')]
-            self.command += ['--scan-build-path', '../llvm-project/clang/tools/scan-build/bin/scan-build']  # Only generate results page on the second comparison
-            self.command += ['--delete-results']
-            if CURRENT_HOSTNAME in EWS_BUILD_HOSTNAMES + TESTING_ENVIRONMENT_HOSTNAMES and self.getProperty('github.base.ref', DEFAULT_BRANCH) == DEFAULT_BRANCH:
-                self.command += [
-                    '--builder-name', self.getProperty('buildername', ''),
-                    '--build-number', self.getProperty('buildnumber', ''),
-                    '--buildbot-worker', self.getProperty('workername', ''),
-                    '--buildbot-master', CURRENT_HOSTNAME,
-                    '--report', RESULTS_DB_URL,
-                    '--architecture', self.getProperty('architecture', ''),
-                    '--platform', self.getProperty('platform', ''),
-                    '--version', self.getProperty('os_version', ''),
-                    '--version-name', self.getProperty('os_name', ''),
-                    '--style', self.getProperty('configuration', ''),
-                    '--sdk', self.getProperty('build_version', '')
-                ]
-        else:
-            self.command += ['--check-expectations']
+        self.command += ['--build-output', SCAN_BUILD_OUTPUT_DIR, '--check-expectations']
```

Comment:
Do we need it in separate line or can merge it in `self.command =` ?

Reply:
It can be merged!